### PR TITLE
Create gw_docker_build_docs

### PR DESCRIPTION
**Pull request by mmann1123**
Allows us to build docs on the fly, one command

### CODE REVIEW

File: dockerfiles/gw_docker_build_docs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    git \
+    libgdal-dev \
+    libgl1 \
+    libspatialindex-dev \ 
+    wget \
+    python-is-python3 \
+    pip \
+    g++ 
+
+RUN pip install Cython numpy
```
Comment (jgrss) [inline dockerfiles/gw_docker_build_docs:20]: Do we want/need to pin the versions on anything here?

File: dockerfiles/gw_docker_build_docs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    python-is-python3 \
+    pip \
+    g++ 
+
+RUN pip install Cython numpy  
+RUN pip install git+https://github.com/jgrss/geowombat
+RUN pip install sklearn-xarray  pip-tools rtree ipykernel sphinx graphviz numpydoc sphinx_automodapi sphinxcontrib.bibtex==1.0
+
+RUN python -c "import geowombat as gw"
+RUN python -c "import rasterio;from rasterio.crs import CRS; cc = CRS.from_epsg(4326)"
```
Comment (jgrss) [inline dockerfiles/gw_docker_build_docs:25]: These are just tests, right?

**Review by jgrss — APPROVED** [review]

File: dockerfiles/gw_docker_build_docs (hunk 1 above)
Comment (mmann1123) [reply dockerfiles/gw_docker_build_docs:20]: This build seems to be working. But I differ to you, I don't have a clear sense of how best to handle this.

File: dockerfiles/gw_docker_build_docs (hunk 2 above)
Comment (mmann1123) [reply dockerfiles/gw_docker_build_docs:25]: Yes that's just a basic test to make sure things are set up.

File: dockerfiles/gw_docker_build_docs (hunk 1 above)
Comment (jgrss) [reply dockerfiles/gw_docker_build_docs:23]: It might be a good idea for the Docker files.

**Comment by mmann1123** [comment]
Ok I just added the versions and added jax & jaxlib. I was able to build the image and generate docs.  

Not sure how to interpret the circle i build error

Should I have it automatically branch, commit and create a pull request?

**Comment by jgrss** [comment]
> Ok I just added the versions and added jax & jaxlib. I was able to build the image and generate docs.
> 
> Not sure how to interpret the circle i build error
> 
> Should I have it automatically branch, commit and create a pull request?

I think you need to merge `main` into this branch because this branch doesn't have `.circleci`.

**Comment by jgrss** [comment]
> > Ok I just added the versions and added jax & jaxlib. I was able to build the image and generate docs.
> > Not sure how to interpret the circle i build error
> > Should I have it automatically branch, commit and create a pull request?
> 
> I think you need to merge `main` into this branch because this branch doesn't have `.circleci`.

So I think just do:
```commandline
git checkout main
git pull origin main
git checkout mmann1123-build-docs
git merge main
git push origin mmann1123-build-docs
```

**Review by jgrss — APPROVED** [review]
Looks good. I merged the latest from `main` into this branch. It should be ready to merge whenever.

**Pull request merged**